src/index.js

```diff
@@ -144,8 +150,14 @@ class Dotenv {
       return obj
     }, {})
 
-    // fix in case of missing
-    formatted['process.env'] = '{}'
+    // We have to stub any remaining `process.env`s due to Webpack 5 not polyfilling it anymore
+    // https://github.com/mrsteele/dotenv-webpack/issues/240#issuecomment-710231534
+    // However, if someone targets Node or Electron `process.env` still exists, and should therefore be kept
+    // https://webpack.js.org/configuration/target
+    if (!target.startsWith('node') && !target.startsWith('electron')) {
```

Comment:
Not all `electron` targets are node targets. Some of them are for the browser environment.